cloudnatix/manifest.json

```diff
@@ -0,0 +1,65 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "caebfbe2-48ba-443f-b2c6-bd80122b2605",
+  "app_id": "cloudnatix",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Integration with CloudNatix",
+    "title": "CloudNatix",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "dashboard",
+        "image_url": "images/cloudnatix-dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Cloud",
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Google Cloud",
+      "Category::Kubernetes",
+      "Category::Metrics",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
please include Submitted Data type and/or queried data type as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags)